src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -405,6 +405,16 @@ public virtual bool IsDefaultTupleElement
         {
             get
             {
+                Debug.Assert(!(this is TupleElementFieldSymbol) && !(this is TupleErrorFieldSymbol));
```

Comment:
>Debug.Assert(!(this is TupleElementFieldSymbol) && !(this is TupleErrorFieldSymbol)); [](start = 16, length = 85)

Should these types override this property and `throw ExceptionUtilities.Unreachable;` instead? Same question for property below.